tests/unit-tests/ambiguous/__snapshots__/arrow-parens-avoid.test.ts.snap

```diff
@@ -726,7 +726,7 @@ oops => {};
 exports[`ambiguous > config > arrowParens: "avoid" > (oops) => {} > with semi, with newline > it formats ../cases/gjs/simple.gjs 1`] = `
 "<template>
   what
-</template>
+</template>;
```

Comment:
I think this is how prettier would handle it if template is transformed to function invocation